apps/monitor-client/lib/features/CollapsedView.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { AppContext, IAppContext } from '../context';
+import TableCollapsible from '../components/UsersTable';
+
+export function CollapsedView() {
```

Comment:
CollapsedView of what?
this is users feature? or this is reusable component?
in first case move it to the /features/users/components/
in second lib/components